packages/account-v2/src/containers/PersonalDetailsForm/validation.ts

```diff
@@ -0,0 +1,151 @@
+import * as Yup from 'yup';
+import { GetSettings, ResidenceList, StatesList } from '@deriv/api-types';
+
+type initialValues = {
+    addressCity?: string;
+    addressLine1?: string;
+    addressLine2?: string;
+    addressPostcode?: string;
+    addressState?: string;
+    dateOfBirth?: number | null;
+    email?: string;
+    emailConsent?: 0 | 1;
+    employmentStatus?: string;
+    firstName?: string;
+    lastName?: string;
+    phone?: string | null;
+    placeOfBirth?: string;
+    requestProfessionalStatus?: 1;
+    residence?: string | null;
+    taxIdentificationNumber?: string;
+};
+
+const min2max50text = 'You should enter 2-50 characters.';
+const lessThan70Text = 'Should be less than 70.';
+
+const getBaseSchema = () =>
```

Comment:
Can u check if u can make use of this - `packages/account-v2/src/utils/personal-details-utils.ts`. Since I had necessity for some part of Personal details for IDV, I had created a schema